tests/ui-api-layer-acceptance-tests/domain/remoteenvironment/remoteenvironment_test.go

```diff
@@ -0,0 +1,114 @@
+// +build acceptance
+
+package remoteenvironment
+
+import (
+	"testing"
+	"github.com/kyma-project/kyma/tests/ui-api-layer-acceptance-tests/dex"
+	"github.com/kyma-project/kyma/tests/ui-api-layer-acceptance-tests/graphql"
+	"github.com/stretchr/testify/require"
+	"github.com/magiconair/properties/assert"
+)
+
+type reMutationResponse struct {
+	Name        string            `json:"name"`
+	Description string            `json:"description"`
+	Labels      map[string]string `json:"labels"`
+}
+
+type deleteReResponse struct {
+	Name string `json:"name"`
+}
+
+func TestRemoteEnvironmentMutations(t *testing.T) {
+	if dex.IsSCIEnabled() {
+		t.Skip("SCI Enabled")
+	}
+	c, err := graphql.New()
+	require.NoError(t, err)
+
+	const fixName = "test-ui-api-re"
+	expectedResponse := reMutationResponse{
+		Name:        fixName,
+		Description: "fix-desc1",
+		Labels: map[string]string{
+			"fix": "lab",
+		},
+	}
+	t.Log("Create RemoteEnvironment")
+	resp, err := createRE(c, expectedResponse)
```

Comment:
it's really strange when you are reading it `createRE(c, expectedResponse)`, just name it `createRE(c, fixResponse)`